src/package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "src",
+    "description": "React Native WebView component for iOS, Android, macOS, and Windows",
+    "main": "../index.js",
+    "typings": "../index.d.ts",
+    "author": "Jamon Holmgren <jamon@infinite.red>",
+    "contributors": [
+      "Thibault Malbranche <malbranche.thibault@gmail.com>"
+    ],
+    "license": "MIT",
+    "version": "11.3.2",
+    "private": true,
+    "homepage": "https://github.com/react-native-webview/react-native-webview#readme",
+    "scripts": {
+      "start": "node node_modules/react-native/local-cli/cli.js start",
+      "start:android": "react-native run-android",
+      "start:ios": "react-native run-ios",
+      "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos",
+      "start:windows": "react-native start --use-react-native-windows",
+      "ci": "CI=true && yarn lint",
+      "ci:publish": "yarn semantic-release",
+      "lint": "yarn tsc --noEmit && yarn eslint ./src --ext .ts,.tsx",
+      "build": "yarn tsc",
+      "prepare": "yarn build",
+      "appium": "appium",
+      "test:windows": "yarn jest --setupFiles=./jest-setups/jest.setup.windows.js"
+    },
+    "rn-docs": {
+      "title": "Webview",
+      "type": "Component"
+    },
+    "peerDependencies": {
+      "react-native": ">=0.60 <0.64"
+    },
+    "dependencies": {
+      "escape-string-regexp": "2.0.0",
+      "invariant": "2.2.4"
+    },
+    "devDependencies": {
+      "@babel/core": "7.4.5",
+      "@babel/runtime": "7.4.5",
+      "@react-native-community/cli": "^4.8.0",
+      "@react-native-community/cli-platform-android": "^4.8.0",
+      "@react-native-community/cli-platform-ios": "^4.8.0",
+      "@semantic-release/git": "7.0.16",
+      "@types/invariant": "^2.2.30",
+      "@types/jest": "24.0.18",
+      "@types/react": "16.9.34",
+      "@types/react-native": "0.62.5",
+      "@types/selenium-webdriver": "4.0.9",
+      "@typescript-eslint/eslint-plugin": "2.1.0",
+      "@typescript-eslint/parser": "2.1.0",
+      "appium": "1.17.0",
+      "babel-eslint": "10.0.3",
+      "babel-jest": "24.8.0",
+      "babel-plugin-module-resolver": "3.1.3",
+      "eslint": "6.3.0",
+      "eslint-config-airbnb": "18.0.1",
+      "eslint-config-prettier": "6.2.0",
+      "eslint-plugin-import": "2.18.2",
+      "eslint-plugin-jsx-a11y": "6.2.3",
+      "eslint-plugin-react": "7.14.3",
+      "eslint-plugin-react-native": "3.7.0",
+      "jest": "24.9.0",
+      "metro": "^0.56.4",
+      "metro-react-native-babel-preset": "^0.59.0",
+      "react": "16.11.0",
+      "react-native": "0.62.2",
+      "react-native-macos": "0.60.0-microsoft.73",
+      "react-native-windows": "0.62.17",
+      "selenium-appium": "0.0.15",
+      "selenium-webdriver": "4.0.0-alpha.7",
```

Comment:
if the tests are just using the example app then maybe it'll be worth moving these to the example